client/src/app/tabs/EngineProfile.js

```diff
@@ -322,12 +312,12 @@ function Link(props) {
 function filterEngineOptions() {
 
   if (!Flags.get(DISABLE_PLATFORM) && ! Flags.get(DISABLE_ZEEBE))
-    return engineProfiles;
+    return ENGINE_PROFILES;
 
-  return engineProfiles.filter(
+  return ENGINE_PROFILES.filter(
     option => (
-      Flags.get(DISABLE_PLATFORM) && option.executionPlatform != 'Camunda Platform' ||
-      Flags.get(DISABLE_ZEEBE) && option.executionPlatform != 'Camunda Cloud'
+      Flags.get(DISABLE_PLATFORM) && option.executionPlatform != ENGINES.PLATFORM ||
+      Flags.get(DISABLE_ZEEBE) && option.executionPlatform != ENGINES.CLOUD
```

Comment:
Let's use strict comparison here:

```suggestion
      Flags.get(DISABLE_PLATFORM) && option.executionPlatform !== ENGINES.PLATFORM ||
      Flags.get(DISABLE_ZEEBE) && option.executionPlatform !== ENGINES.CLOUD
```